apps/voting/contracts/Voting.sol

```diff
@@ -422,8 +580,73 @@ contract Voting is IForwarder, AragonApp {
         emit ExecuteVote(_voteId);
     }
 
+    /**
+     * @dev internal function to add the `_voter` to the list of delegated voters for the `_delegate` and update the delegate for the `_voter`
+     * @param _delegate address of the delegate
+     * @param _voter address of the voter
+     */
+    function _addDelegatedAddressFor(address _delegate, address _voter) internal {
+        uint256 delegatedVotersCount = delegatedVoters[_delegate].addresses.length;
+        require(delegatedVotersCount <= UINT_96_MAX, ERROR_MAX_DELEGATED_VOTERS_REACHED);
+
+        delegatedVoters[_delegate].addresses.push(_voter);
+        delegates[_voter] = Delegate(_delegate, uint96(delegatedVotersCount));
```

Comment:
I think we can consider moving mapping update to the `_removeDelegatedAddressFor `, it will make these functions more similar, and it's not a big deal in terms of gas cost (+100).